ynput/OpenPype
# 3dsmax: Publishing Deadline jobs from RedShift

## Changelog Description
Fix the bug of being uable to publish deadline jobs from Redshift
Use Current File instead of Published Scene for just Redshift.
1. add save scene before rendering to ensure the scene is saved after the modification.
2. add separated aov files option to allow users to choose to have aovs in render output
3. add validator for render publish to aovid overriding the previous renders
![image](https://github.com/ynput/OpenPype/assets/64118225/7473ff66-485e-481d-ac4f-70eab47198c7)

## Additional info
**Please delete the old instance and create the new one for the test.**
1. 3dsMax doesn't support OCIO config yet so the related parameter is hardcorded.
2. Max in Redshift does support OCIO and currently hardcoded the data seen from RS in the collector.
5.`rt.Redshift_Renderer().SeparateAovFiles` seems to be broken as it doesn't bind to the render setting when executing the code. So currently please make sure you dont check `Separate AOV Files`. 
![image](https://github.com/ynput/OpenPype/assets/64118225/78b13b9a-d73b-4f3b-884d-da6e8cef1352)

## Testing notes:
1. Launch 3dsmax via Openpype
6. Use Redshift as Renderer
7. Create Render Instance
8. Publish Render
9. It should only includes MaxRenderMain.exr and Cryptomatte.exr along with metadata in the related output folder
